system/jlib/jlzw.cpp

```diff
@@ -2484,6 +2484,19 @@ bool isCompressedFile(IFileIO *iFileIO, CompressedFileTrailer *trailer=nullptr)
                     return true;
             }
         }
+        //If the file is empty, but we are expecting a compressed file, fill in the trailer with default information
+        if ((fsize == 0) && trailer)
```

Comment:
maybe this could be an "else if" for clarity (that not going to enter this block if (fsize>=sizeof(WinCompressedFileTrailer) ?